server/data/authClientBuilder.js

```diff
@@ -3,6 +3,13 @@ const Agent = require('agentkeepalive')
 const { HttpsAgent } = require('agentkeepalive')
 const logger = require('../../log')
 const config = require('../../config')
+const csRedis = require('cache-service-redis')
+const redisCache = new csRedis({
+  port: config.redis.port,
+  hostname: config.redis.port,
+  auth: config.redis.password
```

Comment:
When testing, if you add `verbose: true` here you will see the cache output in your console.